### PR TITLE
snap: refresh sources to `noble`

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -111,15 +111,15 @@ parts:
...
+    source-branch: applied/ubuntu/noble-devel
     source-depth: 1
 
   neatvnc:
     after: [ffmpeg]
     plugin: nil
     source: https://git.launchpad.net/ubuntu/+source/neatvnc
     source-type: git
-    source-branch: applied/ubuntu/mantic-devel
+    source-branch: applied/ubuntu/noble-devel
```
Comment (mattkae) [inline snap/snapcraft.yaml:122]: I think we also need:

```yaml
build-packages:
  ...
  - libxxhash-dev

...
stage-packages:
  ...
  - libxxhash0
```

**Review by mattkae — CHANGES_REQUESTED** [review]
We need another package to fix CI, right?

File: snap/snapcraft.yaml (hunk 1 above)
Comment (AlanGriffiths) [reply snap/snapcraft.yaml:122]: `apt show libxxhash-dev` says it contains a static library so, maybe, there's no need for the shared library

Comment (Saviq) [reply snap/snapcraft.yaml:122]: > `apt show libxxhash-dev` says it contains a static library so, maybe, there's no need for the shared library

Indeed, seems to work without.

**Review by mattkae — APPROVED** [review]
:+1:

**Pull request merged**